src/csharp/Grpc.Core/AsyncUnaryCall.cs

```diff
@@ -85,6 +85,16 @@ public TaskAwaiter<TResponse> GetAwaiter()
         {
             return responseAsync.GetAwaiter();
         }
+        
+        /// <summary>
+        /// Configures an awaiter used to await this object.
+        /// </summary>
+        /// <param name="continueOnCapturedContext">true to attempt to marshal the continuation back to the original context captured;
+        /// otherwise, false.</param>
+        public ConfiguredTaskAwaitable ConfigureAwait(bool continueOnCapturedContext)
+        {
+            return responseAsync.ConfigureAwait(continueOnCapturedContext);
```

Comment:
IF we do this, it would need to be done for other types of call as well (client streaming, etc.).